google/cloud/spanner_v1/streamed.py

```diff
@@ -143,6 +145,11 @@ def __iter__(self):
             while iter_rows:
                 yield iter_rows.pop(0)
 
+    def to_dataframe(self):
+        """Returns the response in a pandas dataframe of the StreamedResultSet object"""
```

Comment:
You'll need to show types in a structured way. See `rtype` in other docstrings.

Reply:
Done. Linked to pandas Dataframe docs